Components/FlightControl/FlightTask.cpp

```diff
@@ -42,7 +43,7 @@ void FlightTask::InitTask()
  */
 void FlightTask::Run(void * pvParams)
 {
-    uint32_t tempSecondCounter = 0; // TODO: Temporary counter, would normally be in HeartBeat task or HID Task, unless FlightTask is the HeartBeat task
+    //uint32_t tempSecondCounter = 0; // TODO: Temporary counter, would normally be in HeartBeat task or HID Task, unless FlightTask is the HeartBeat task
```

Comment:
nit: Don't comment out, just remove. The whole point of version control is that we can see previous changes and add things back if needed.